revops/index.html

```diff
@@ -15,6 +15,23 @@
 <link href="/css/revops.css" rel="stylesheet" type="text/css" />
 <link href="/css/homestyle.css" rel="stylesheet" type="text/css" />
 
+<style>
+  .who-we-are-section {
+    grid-template-columns: auto 1fr 480px;
+  }
+  .who-we-are-section .team-photos {
+    grid-template-columns: 150px 150px 150px;
+  }
+  @media screen and (max-width: 1100px) {
+    .who-we-are-section {
+      grid-template-columns: auto 1fr 210px;
+    }
+    .who-we-are-section .team-photos {
+      grid-template-columns: 90px 90px;
+    }
+  }
+</style>
```

Comment:
Is there any reason for the style tag here? Why is the styling not in a CSS file?

Reply:
It was my workaround for changing the styles coming from the home.scss, for the revops page only.

Reply:
Fixed in 85926a1c8b209d8fe3c32548262f2eb565248ace